Formula/c/crystal.rb

```diff
@@ -40,6 +40,7 @@ class Crystal < Formula
   end
 
   depends_on "bdw-gc"
+  depends_on "gcc"
```

Comment:
If this is only going to be used in the linux version, then this line should be guarded by `if OS.linux?`